datapusher/setup/datapusher_settings.py

```diff
@@ -0,0 +1,29 @@
+import uuid
+
+DEBUG = False
+TESTING = False
+SECRET_KEY = str(uuid.uuid4())
+USERNAME = str(uuid.uuid4())
+PASSWORD = str(uuid.uuid4())
+
+NAME = 'datapusher'
+
+# database
+
+SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/job_store.db'
```

Comment:
Should we connect it to `db` service instead? Just a question, I'm not sure it adds any benefits...

Reply:
This is the variable in base image. It's later overridden `SQLALCHEMY_URL` I think. But I'm not sure neither tbh